docs/admin-manual/manual.md

```diff
@@ -263,6 +263,43 @@ The download feature in the file browser can be disabled separately with the fol
       # Port where HiveServer2 Thrift server runs on.
       ## hive_server_port=10000
 
+### LLAP
```

Comment:
Note: this doc content should probably moved to https://github.com/cloudera/hue/blob/master/docs/docs-site/content/administrator/configuration/editor/_index.md#hive (as the real doc website is now there)

Reply:
https://review.cloudera.org/r/14066/